### PR TITLE
Extract partial templates

### LINKED ISSUE #17: Extract common templates for item views
## Description

A lot of headers and navigation elements are common to all items so we should consider extracting them into partial templates.

### LINKED ISSUE #18: RIP Google+
## Description

We still have metadata for Google+ in our items and index views. We should remove it.

### PR DESCRIPTION
**Pull request by gfiorav**
Este PR extrae algunas cosas comunes de todos los temaplates.

Fixes https://github.com/beachboyslegacy/beachboyslegacy.github.com/issues/17
Fixes https://github.com/beachboyslegacy/beachboyslegacy.github.com/issues/18

Estos son los nuevos partials:
```
common_headers.html.jinja2
footers.html.jinja2
item_details_header.html.jinja2
post_scripts.html.jinja2
```

También he aprovechado a quitar lineas en blanco/unificarlas en todos los templates.

### CODE REVIEW

File: templates/partials/item_details_header.html.jinja2 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div class="item-details-header">
+  <!-- <a id="open-overlay-menu" class="header__menu-icon" href="javascript:void(0)" onClick="ga('send', 'event', 'Navigation', 'click', 'Overlay Menu')"><webicon icon="material:menu"/></a> -->
```
Comment (carlos-sanchez) [inline templates/partials/item_details_header.html.jinja2]: This link can be removed

File: templates/partials/footers.html.jinja2 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="mobile-footer">
```
Comment (carlos-sanchez) [inline templates/partials/footers.html.jinja2:1]: The div is missing the closing </div> tag

File: templates/partials/common_headers.html.jinja2 [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+<meta name="twitter:image" content="https://beachboyslegacy.com/{{item["parent"]["image"]}}" />
+<meta name="twitter:card" content="summary_large_image" />
+<meta name="twitter:site" content="@beachboyslegacy">
+<meta name="twitter:image:alt" content="Beach Boys Legacy">
+
+
+<!-- For Google+ -->
+<!-- <html itemscope itemtype="http://schema.org/Article"> -->
+
+<meta itemprop="image" content="https://beachboyslegacy.com/{{item["parent"]["image"]}}">
```
Comment (carlos-sanchez) [inline templates/partials/common_headers.html.jinja2]: The Google+ metas can be removed

File: templates/partials/common_headers.html.jinja2 [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    },
+    "button": {
+      "background": "#f1d600"
+    }
+  },
+  "content": {
+    "href": "cookies-policy.html"
+  }
+})});
+</script>
```
Comment (carlos-sanchez) [inline templates/partials/common_headers.html.jinja2]: We are already calling the cookies plugin from post_scripts so I think it can be removed from here

Comment (gfiorav) [reply templates/partials/common_headers.html.jinja2]: I'll move it to post scripts.

File: templates/partials/footers.html.jinja2 (hunk 2 above)
Comment (gfiorav) [reply templates/partials/footers.html.jinja2:1]: good catch! I added it to the partial

**Pull request merged**